config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
 
-  get 'oauths/oauth'   # is it really need fir us?
-
+  get 'oauths/oauth'   # is it really need for us?
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.